packages/fluentui/react-northstar/src/components/Chat/ChatMessage.tsx

```diff
@@ -452,7 +458,7 @@ export const ChatMessage: ComponentWithAs<'div', ChatMessageProps> &
 
   const authorElement = Text.create(author, {
     defaultProps: () => ({
-      size: 'small',
+      ...(!compact && { size: 'small' }),
```

Comment:
nit: This is personal preference, feel free to ignore
```suggestion
      size: compact ? undefined : 'small'
```

Reply:
Done.